lib/pluginHarvest.js

```diff
@@ -35,7 +35,10 @@ exports.register = function( server, pluginOptions, next ) {
   } );
 
   server.expose( 'putBasket', function( options ) {
-    return putBasket( Harvest, harvestDb, options );
+    return putBasket( Harvest, harvestDb, options )
+    .fail(() => {
```

Comment:
not really, if the put fails and then the post call fails after wards the-works will log the error in the controller